src/scripts/main.js

```diff
@@ -18,10 +18,31 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+let tree = document.querySelector('#tree');
+
+if (!tree) {
+  tree = document.createElement('ul');
+  tree.id = 'tree';
+  document.body.appendChild(tree);
+}
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key in data) {
```

Comment:
Avoid using 'for-in' loop to iterate over an object unless you also include a hasOwnProperty check. It's recommended to use Object.keys() or Object.entries() to iterate through objects. It's also a good practice to check if the object is not null or undefined before iterating over it.